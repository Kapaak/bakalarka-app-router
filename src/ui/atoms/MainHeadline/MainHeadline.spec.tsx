import { render } from "@testing-library/react";

import { MainHeadline } from "./MainHeadline";

describe("MainHeadline", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<MainHeadline />);

    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
