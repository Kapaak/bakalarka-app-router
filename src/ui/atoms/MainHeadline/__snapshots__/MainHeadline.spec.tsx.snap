// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainHeadline should render successfully 1`] = `
<body>
  <div>
    <h1
      class="text-4xl lg:text-5xl lg:leading-[1.1] undefined"
    />
  </div>
</body>
`;
